### ISSUE TITLE
wunderfixer fails to work correctly for today for stations in east of Greenwich when local date is ahead of the GMT date.

### ISSUE THREAD

**Issue by rodyager** [weewx/weewx#445]
There is a problem with the WU API which only affects stations that lie east of the Greenwich meridian, and only when the date to be "fixed" is today's date and only while the local date is ahead of the GMT date. (eg. I am in Sydney Australia, which is 10 hours ahead of GMT so this problem only occurs between midnight and 10am local time for me.)

A patch for wunderfixer (development branch) to fix this is attached. It uses the 1day API when the date to be "fixed" is today, because this API doesn't suffer from the above issue. 

Rod

[wunderfixer.txt](https://github.com/weewx/weewx/files/3624464/wunderfixer.txt)

**Comment by tkeffer** [comment]
Thanks, Rod! Fixed in commit https://github.com/weewx/weewx/commit/d1c3abee6f7357119276907b0de8d949a71587dc

**Comment by rodyager** [comment]
It looks like I took long to submit this patch as WU has now fixed the behaviour of their API for stations east of Greenwich, and the patch is no longer necessary.

You should revert the change in the interests of code simplicity and readability.

Sorry for this bother.

Rod


On 19 Sep 2019, at 12:54 am, Tom Keffer <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks, Rod! Fixed in commit d1c3abe<https://github.com/weewx/weewx/commit/d1c3abee6f7357119276907b0de8d949a71587dc>

**Comment by tkeffer** [comment]
Really? If you're sure, I'll revert the change.

**Comment by rodyager** [comment]
I’d been running this patch for quite a while, and duly applied it to my local copy when I upgraded to 4.0.8a yesterday. That was what prompted me to submit it.

It was only after I had submitted it that it ocurred to me that I should perform the sanity check of verifying that it was still required. (The history was that WU had more substantial problems with this API that were fixed, leaving just the case of stations east of Greenwich that had already transitioned to the next day as the one edge case they had not addressed. This morning’s testing shows that this has now also been addressed.)

Rod

On 19 Sep 2019, at 7:39 am, Tom Keffer <notifications@github.com<mailto:notifications@github.com>> wrote:


Really? If you're sure, I'll revert the change.
